content/_includes/icons/audio.svg

```diff
@@ -3,8 +3,6 @@
   xmlns="http://www.w3.org/2000/svg"
   aria-hidden="true"
   focusable="false"
-  width="20px"
-  height="20px"
```

Comment:
@mirisuzanne So. Herman runs these through SVGO, which by default removes the `viewBox` attr if it matches the height/width exactly. Then we run those optimized SVGs through gulp-svg-symbols, which breaks (with our custom sprite-sheet, which relies on `viewBox`) if `viewBox` is missing...

I'm not clear on the tradeoffs here, or when we do or don't need height/width. Probably this is a bug that should be fixed in Herman -- let's discuss.

Reply:
well, they don't do exactly the same thing - but they do overlap a lot.

- `viewBox` is better for most things. We don't ever want to remove it. if we're choosing, it should absolutely be what we keep.
- I like _also_ having height/width, because they provide a default size for the icon. It's less essential, but it means they'll remain reasonably-sized until we explicitly resize them in CSS (which is simple when we want it).

You can see the difference here: https://cdpn.io/mirisuzanne/debug/002175ac77b34d172f1cdf769673c899

So… I'm ok with this, and it shouldn't hold up the  branch, but I don't love it as a restriction. Long term, I'm not sure we should have Herman do so much "optimization" as a documentation tool.

Reply:
Yeah, that makes sense to me. It'd be a quick fix to Herman to stop optimizing SVGs entirely...